modules/cnv-viewing-virtual-machine-logs-web.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-logs-events.adoc
+
+[id="cnv-viewing-virtual-machine-logs-web_{context}"]
+= Viewing virtual machine logs in the web console
+
+Get virtual machine logs from the associated virtual machine launcher pod.
+
+.Procedure
+
+. In the {ProductName} console, click *Workloads* -> *Virtual Machines*.
+. Click the virtual machine to open the *Virtual Machine Details* screen.
```

Comment:
Not "screen," most likely.

> ! screen n
Do not use “panel,” “screen,” and “window” interchangeably because they are not synonyms. A “screen” is
the physical surface of a display device, not the information that is displayed.
A “panel” is an area of a screen that displays formatted information and can also include entry fields. A
“window” is an area of the screen with visible boundaries in which an application program or information is
displayed or in which a dialog is presented.

Reply:
Panel it is